tests/test_pyseq.py

```diff
@@ -487,6 +489,40 @@ def test_get_sequences_is_working_properly_3(self):
                 seq.format('%h%p%t %r')
             )
 
+    def test_walk_succeed(self):
```

Comment:
btw, I removed this test before merging because I also removed the dir1 and dir2 folders because I didn't want the test folder to start getting too cluttered. I'll have to rewrite this test eventually, and it probably makes sense to do this programmatically rather than check in a bunch of dummy files to the repo.